src/main/resources/hudson/plugins/parameterizedtrigger/BuildInfoExporterAction/summary.groovy

```diff
@@ -1,5 +1,8 @@
 package hudson.plugins.parameterizedtrigger.BuildInfoExporterAction
 
+if (System.getProperty("ptp.disableActionViews") != null) {
```

Comment:
Considering that this check is done four times, how about extracting the logic into a boolean static helper method, maybe in ParameterizedTriggerUtils?

Reply:
I considered it, but decided against it since it would increase number of lines with approx 50% without increasing readability. Though I will lift out the property key to a constant to make adjustments easier.